bug_change_status_page.php

```diff
@@ -291,7 +291,7 @@
 		<td>
 <?php
 			if( $t_has_write_access ) {
-				print_custom_field_input( $t_def, $f_bug_id );
+				print_custom_field_input( $t_def, $f_bug_id, $t_require );
```

Comment:
$t_require = $t_def['require_' . $t_custom_status_label];
This would usually contain a string "0" or "1"
Should we cast here to a bool?
Or should the definition array contain proper booleans already?